database/cassandra/cassandra.go

```diff
@@ -122,17 +119,15 @@ func (c *Cassandra) Open(url string) (database.Driver, error) {
 		cluster.Timeout = timeout
 	}
 
-	c.session, err = cluster.CreateSession()
-
+	session, err := cluster.CreateSession()
 	if err != nil {
 		return nil, err
 	}
 
-	if err := c.ensureVersionTable(); err != nil {
-		return nil, err
-	}
-
-	return c, nil
+	return WithInstance(session, &Config{
+		KeyspaceName:    strings.TrimPrefix(u.Path, "/"),
```

Comment:
Is the expectation that the config KeyspaceName not have `/` as a prefix?
I'm worried about this being a backwards incompatible change.

It looks like Cassandra doesn't allow `/` in the keyspace name: https://docs.datastax.com/en/cql/3.3/cql/cql_reference/cqlCreateKeyspace.html

Reply:
NVM, I was worried about existing instances of Config, but turns out that's not a problem since `WithInstance()` was recently added.